tests/test_views.py

```diff
@@ -51,7 +50,9 @@ def test_query_matching_authors(self):
         # Create ten citations to match the query and one that doesn't.
         factories.CitationFactory.create_batch(10, author='Verne, Jules')
         factories.CitationFactory(author='Dan Longshot')
```

Comment:
Since authors view should return any authors that _start with_ the value given in the URL, would it be sensible to also create a Citation with author like 'Verne, Busta' or something here to make sure that also matches? @somexpert @damonkelley is that too much?